src/Compilers/VisualBasic/Test/Semantic/Semantics/Conversions.vb

```diff
@@ -605,7 +605,8 @@ End Class
                     New TypeAndValue(uint64Type, CULng(18)),
                     New TypeAndValue(decimalType, CDec(-11.3)),
                     New TypeAndValue(doubleType, CDbl(&HF000000000000000UL)),
-                    New TypeAndValue(doubleType, CDbl(&H70000000000000F0L)),
```

Comment:
>New TypeAndValue(doubleType, CDbl(&H70000000000000F0L)), [](http://example.com/codeflow?start=20&length=56)

Please keep this value

Reply:
I don't think we should keep `CDbl(&H70000000000000F0L)` because it's misleading: the integer `&H70000000000000F0L` can't be represented exactly by a Double. As I noted in the PR description, CDbl rounds it down to `&H7000000000000000L`, which is why it failed to catch this bug.

Reply:
Thanks for the explanation. Sounds reasonable.